Publishing TAP 1.3 release doc updates (#118)
aws-quickstart/quickstart-vmware-tanzu-application-platform: docs/deployment_guide/partner_editable/overview.adoc

> This Quick Start deploys {partner-product-name} on the AWS Cloud. This Quick Start is for users who want a repeatable and customizable reference deployment for https://tanzu.vmware.com/application-platform[{partner-product-name}] on Amazon Elastic Kubernetes Service (Amazon EKS) using AWS CloudFormation. This guide provides instructions for deploying the {partner-product-short-name} to Amazon EKS clusters.

Reviewer: ```suggestion
This Quick Start deploys {partner-product-name} to the AWS Cloud. This Quick Start is for users who want a repeatable and customizable reference deployment for https://tanzu.vmware.com/application-platform[{partner-product-name}] on Amazon Elastic Kubernetes Service (Amazon EKS) using AWS CloudFormation. This guide provides instructions for deploying the {partner-product-short-name} to Amazon EKS clusters.
```

> This Quick Start deploys {partner-product-name} on the AWS Cloud. This Quick Start is for users who want a repeatable and customizable reference deployment for https://tanzu.vmware.com/application-platform[{partner-product-name}] on Amazon Elastic Kubernetes Service (Amazon EKS) using AWS CloudFormation. This guide provides instructions for deploying the {partner-product-short-name} to Amazon EKS clusters.

Reviewer: ```suggestion
This Quick Start deploys {partner-product-name} on the AWS Cloud. This Quick Start is for users who want a repeatable and customizable reference deployment for https://tanzu.vmware.com/application-platform[{partner-product-name}^] on Amazon Elastic Kubernetes Service (Amazon EKS) using AWS CloudFormation. This guide provides instructions for deploying the {partner-product-short-name} to Amazon EKS clusters.
```

> To learn more about {partner-product-short-name}, please visit https://tanzu.vmware.com/application-platform[https://tanzu.vmware.com/application-platform].

Reviewer: ```suggestion
For more information about {partner-product-short-name}, refer to https://tanzu.vmware.com/application-platform[VMware Tanzu Application Platform^].
```
